.github/workflows/ci.yml

```diff
@@ -27,8 +27,9 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 2 # required by Scrutinizer
-      - run: composer install --no-progress --no-ansi --no-interaction --dev --prefer-dist
-      - run: composer test && composer inspect
+      - run: composer update --no-interaction --dev --prefer-dist
+      - run: composer run test
+      - run: composer run inspect
```

Comment:
These are Composer scripts, not Composer commands.

Reply:
There is no difference.

If you type `composer`, the scripts are literally listed under "available commands" along with everything else.

Reply:
also, does this actually work, with separate `run` entries? (it should fail the build if either of these commands fail)

not saying it won't, just asking - I only recently started using GH actions so I just stuck with what I knew would work.

if you're confident it will work with separate commands, that's fine. 🙂

Reply:
I am not a software writer. I only care about underlying technologies.
This happened to others.
e.g. https://github.com/PrinsFrank/standards/commit/0f571449ad7821bd97e683ae88f5dfbf1feff0ac

Reply:
_Explicit is better than implicit_ - This is one of my policies.

Reply:
> I am not a software writer. I only care about these underlying technologies. This happened to others. e.g. [PrinsFrank/standards@0f57144](https://github.com/PrinsFrank/standards/commit/0f571449ad7821bd97e683ae88f5dfbf1feff0ac)

Sorry, I don't know what this is, or what it has to do with composer?

> Explicit is better than implicit 

Ah, so the explicit `run` command is to avoid a breaking change in case Composer itself introduces a `test` or `inspect` command?

Okay, that makes sense. 🙂👍

Reply:
> Sorry, I don't know what this is, or what it has to do with composer?

In the above mentioned commit it turned out that there is a typo in the Composer script name.
(after introducing `composer run <script-name>`)